streams/integration-tests/src/test/java/org/apache/kafka/streams/integration/SmokeTestDriverIntegrationTest.java

```diff
@@ -99,8 +99,8 @@ SmokeTestDriver.VerificationResult result() {
     // We set 2 timeout condition to fail the test before passing the verification:
     // (1) 10 min timeout, (2) 30 tries of polling without getting any data
     @ParameterizedTest
-    @CsvSource({"false, false", "true, false"})
-    public void shouldWorkWithRebalance(final boolean stateUpdaterEnabled, final boolean processingThreadsEnabled) throws InterruptedException {
+    @CsvSource({"true, false"})
```

Comment:
This does not look right!
It should be
```suggestion
    @CsvSource({"true", "false"})
```
You can see it when you run the test. It should have two runs -- one for processing threads enabled and one for processing threads disabled, but it only has one with processing threads enabled.
With my suggestion, you get the two runs.

Reply:
Now you might be even able to use `@ValueSource(booleans = {true, false})` instead of `@CsvSource`.